dtreeviz/colors.py

```diff
@@ -31,6 +31,17 @@
 ]
 
 
+mpl_colors = [
+    None,  # 0 classes
+    None,  # 1 class
+]
+for n_classes in range(2, 11):
+    class_colors = []
+    for i in range(0, n_classes):
+        class_colors.append(f'C{i}')
+    mpl_colors.append(class_colors)
+
```

Comment:
would it be more (or less) clear if we did:

```
mpl_colors = [
    None,  # 0 classes
    None,  # 1 class
] + [[f'C{i}' for i in range(0, n_classes)]
              for n_classes in range(2, 11)]
```

Reply:
Looks fine to me!